### ISSUE TITLE
ENA-189 Implement "go to procedure definition"

### ISSUE THREAD

**Issue by rachel-fenichel** [LLK/scratch-blocks#1136]
### Expected Behavior

The "define" option in the context menu on a custom command block should bring the procedure definition into view.

In Scratch 2.0 it just brings it into view--doesn't center it.  We may want to center and/or highlight it.
We have to decide on text for the context menu option.  In Scratch 2.0 it's 'define', which is not very clear.

**Comment by towerofnix** [comment]
"Jump to definition" is a common wording - if a user knows what it does in Scratch, they'll almost immediately know what it does in other IDEs (and vice versa). I'm not sure it has a very obvious meaning to somebody who's never encountered an action like it before, though. (At least it's better than define!)

**Comment by mrjacobbloom** [comment]
Yeah I'm constantly confusing "Define" and "Edit" in the proc context menus because both words could apply to both functionalities. Changing to something more descriptive like "Jump to definition" would help with that

**Comment by rachel-fenichel** [comment]
This is also a feature that we want in Blockly.

**Comment by rachel-fenichel** [comment]
Implemented in Blockly: https://github.com/google/blockly/pull/1763
